network.c

```diff
@@ -1402,6 +1402,30 @@ struct iio_context * network_create_context(const char *host)
 #endif
 	{
 		ret = getaddrinfo(host, IIOD_PORT_STR, &hints, &res);
+#ifdef HAVE_AVAHI
+		/*
```

Comment:
to keep network.c free of #ifdef HAVE_... should we just add a dnssd_resolve_host() function for all dnssd implementations (which is null is some) and just make everything common in network.c?

Reply:
That looks good to me...